specs/scoring_spec.rb

```diff
@@ -41,22 +44,49 @@
   end
 
   describe 'highest_score_from' do
+
     it 'returns nil if no words were passed' do
+      #arrange
+      words = []
+      #act & #assert
+     Scrabble::Scoring.highest_score_from(words).must_be_nil
+
     end
 
     it 'returns the only word in a length-1 array' do
+      words = []
+      words << "pearl"
+      Scrabble::Scoring.highest_score_from(words).must_equal "pearl"
     end
 
     it 'returns the highest word if there are two words' do
+      words = []
+      words.push("zebra", "otter")
```

Comment:
I suggest also varying the order of the words since that makes a difference in a tie.